src/Components/Reports/SystemsPDFReport.js

```diff
@@ -47,32 +21,27 @@ const generateSystemsPDFReport = async (filters, orderBy, orderHow) => {
     const systemsResponse = await fetchSystems(fetchSystemParams);
     const pdfData = formatData(systemsResponse.data, 'pdf');
 
-    const systemsRows = buildSystemsRows(pdfData);
-
-    // description text
+    // first page description and data
     const totalSystems = systemsResponse?.meta?.count;
     const filterText = generateFilterText(filters);
+    const firstPageData =  pdfData.splice(0, systemsReportFirstPage);
+
+    const firstPage = <SystemsFirstPage
+        data={firstPageData}
+        totalSystems={totalSystems}
+        filterText={filterText} />;
+
+    const otherPages = [];
+
+    while (pdfData.length > 1) {
```

Comment:
@PreetiW Just one query - Why are we comparing with `> 1` instead of `> 0`?

Reply:
Good catch @r14chandra, it was by mistake and due to this it would have missed the last record if pdfData had only one element. I have fixed the condition and pushed the code 💯